src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+      class="card__image"
+      src="./images/imac.jpeg"
+      alt="TV APPLE A1419 image
+      ">
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price-block">
+        <div class="card__price-text">
+          Price:
+        </div>
+        <div class="card__price-value">
+          $2,199
+        </div>
+      </div>
+
+      <a href="//"
+      class="card__buy-link card__buy-link--hover"
+      data-qa="hover"
+      >
```

Comment:
same here with code style and tabulation
```suggestion
      <a
        href="#"
        class="card__buy-link card__buy-link--hover"
        data-qa="hover"
      >
```